scripts/train_models.py

```diff
@@ -38,11 +41,13 @@ def train_model(dm: LightningDataModule, experiment_name: str, trainer_config: d
 
 def train_models(dmpair_kwargs: dict, trainer_config: dict, experiment_pair_name: str):
     dmpair = DMPair(**dmpair_kwargs)
+    seed_everything(seed=dmpair_kwargs["seed"])
     train_model(
         dm=dmpair.A,
         experiment_name=f"{experiment_pair_name}_A",
         trainer_config=trainer_config,
     )
+    seed_everything(seed=dmpair_kwargs["seed"])
```

Comment:
should the seeding happen before DMPair creation for control of val split?

Reply:
As far as I can tell no - passing `none` to a datamodule for the `seed` argument will cause an error when `.setup()` is called, even after running `seed_everything`. Looking at the source code it seems the datamodule needs its own seen, and doesn't draw on a global seed at all.

Reply:
ah ok, that makes sense. Good stuff. Were you able to verify that `seed_everything` gives desired consistency for training runs?